README.md

```diff
@@ -25,7 +43,9 @@ What does the following selector do?  `ul.dropdown > li`?
 
 Select 1:
 ```
-[] Selects all li's which are directly inside a ul of class dropdown (children)
+Note: 'ul.dropdown > li' as a selector shows me error. But '.dropdown > li' works. It selects all li's that are directly inside ul's with class dropdown.
```

Comment:
not sure why that gets an error... it works for me in Safari... but good job testing it! :)